lib/pacemaker/pcmk_sched_allocate.c

```diff
@@ -2868,12 +2870,30 @@ stage7(pe_working_set_t * data_set)
         }
     }
 
-    LogNodeActions(data_set, FALSE);
+    /* stage7 only ever outputs to the log, so ignore whatever output object was
+     * previously set and just log instead.
+     */
+    out = pcmk__new_logger();
+    if (out == NULL) {
+        return FALSE;
+    }
```

Comment:
This previously logged at notice level, so I think we need to set that for this loop

Reply:
Fixed.